x-pack/plugins/apm/public/components/app/service_map/Popover/backend_contents.tsx

```diff
@@ -58,13 +60,26 @@ export function BackendContents({ nodeData }: ContentsProps) {
     >['query'],
   });
 
+  const trackEvent = useUiTracker();
+
   return (
     <>
       <EuiFlexItem>
         <StatsList data={data} isLoading={isLoading} />
       </EuiFlexItem>
       <EuiFlexItem>
-        <EuiButton href={detailsUrl} fill={true}>
+        {/* eslint-disable-next-line @elastic/eui/href-or-on-click*/}
+        <EuiButton
+          href={detailsUrl}
+          fill={true}
+          onClick={() => {
+            trackEvent({
+              app: 'apm',
+              metricType: METRIC_TYPE.CLICK,
+              metric: 'service_map_to_backend_detail',
+            });
+          }}
+        >
```

Comment:
Not that important, but you could do:

```suggestion
        <EuiButton
          fill={true}
          onClick={() => {
            trackEvent({
              app: 'apm',
              metricType: METRIC_TYPE.CLICK,
              metric: 'service_map_to_backend_detail',
            });
            history.push(detailsUrl)
          }}
        >
```